client-app/shared/layout/composables/useNavigations.ts

```diff
@@ -17,7 +15,7 @@ const mainMenuLinks = computed<MenuLink[]>(() =>
       ({
         id: item.id,
         route: item.route,
-        title: globals.i18n!.global.t(item.title),
+        title: globals.i18n.global.t(item.title),
         children: (menuLinkLists.value?.[item.id] || []).map((childrenItem) => ({
           id: childrenItem.url?.split("/").pop(),
           title: childrenItem.title,
```

Comment:
Addition to the above comment: we have translation for parent item, however child item is passed as non-translated. This should be somehow synchronized.